testproxy/services/create-client.js

```diff
@@ -68,18 +70,8 @@ const createClient = ({clientMap}) =>
        */
       Object.entries(
         clientConfig.interfaces['google.bigtable.v2.Bigtable'].methods
-      ).forEach(([k, v]) => {
-        if (k === 'ReadRows') {
-          /*
-          TODO: In the future we should apply this for all methods, but right
-          now doing so results in regressions in the TestSampleRowKeys_Generic_MultiStreams
-          and TestSampleRowKeys_Generic_CloseClient conformance tests that need
-          to be addressed.
```

Comment:
The regressions have been addressed by making a copy of the clientConfig so now this timeout can apply to all methods.